mungegithub/github/github.go

```diff
@@ -274,6 +274,11 @@ type MungeObject struct {
 	Annotations map[string]string //annotations are things you can set yourself.
 }
 
+// Number is short for *obj.Issue.Number.
+func (obj *MungeObject) Number() int {
```

Comment:
I only did it so I could make an interface. But yeah, it'd be a convenient
place to check for nil ;)

On Thu, Aug 18, 2016 at 3:51 PM, Eric Paris <notifications@github.com>
wrote:

> In mungegithub/github/github.go
> <https://github.com/kubernetes/contrib/pull/1563#discussion_r75403616>:
>
> > @@ -274,6 +274,11 @@ type MungeObject struct {
> >  	Annotations map[string]string //annotations are things you can set yourself.
> >  }
> >
> > +// Number is short for *obj.Issue.Number.
> > +func (obj *MungeObject) Number() int {
>
> I feel like an idiot for never making this...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1563/files/aee73d1142f4fa0657f7e5338b23f4e4ee5f2b3c#r75403616>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglitVGwwZyq9YVySku7Hj_hhG43eyks5qhOH_gaJpZM4Jn-Mc>
> .
>